openpype/hosts/blender/api/capture.py

```diff
@@ -107,7 +115,7 @@ def capture(
                 view_context=True,
             )
 
-    return filename
+    return str(filepath) + get_extension_from_image_settings(image_settings)
```

Comment:
```suggestion
    return filepath.with_suffix(get_extension_from_image_settings(image_settings)).as_posix()
```

Reply:
Line too long

Reply:
black will solve that

Reply:
Yes, I will fix this within my own commit.